README.adoc

```diff
@@ -338,3 +338,24 @@ Where :
 
 - SHA1: is the given git SHA1 of the james-project repository to merge.
 - RESULTING_BRANCH: is the branch created when merging.
+
+=== Generating a debian package for James
+
+You can generate a debian package for James by using the following process.
+
+First step, you have to build the Docker image
+
+    $ docker build -t build-james-deb --build-arg ITERATION=1 dockerfiles/packaging/debian/
+
+Where:
+
+- ITERATION is the iteration to give to the package.
```

Comment:
Can you explain what this means? Reading this I understand Where Schtroumpf is the Schtroumpf to give to the package.